ethereum/token_balances/token_balances.sql

```diff
@@ -0,0 +1,202 @@
+
+
+CREATE TABLE sandbox.token_balances_proposal_1(
+   ts timestamptz,
+   address bytea,
+   contract_address bytea,
+   token varchar,
+   rawAmount numeric,
+   amount numeric,
+   usd_amount numeric,
+   PRIMARY KEY( ts,address,contract_address )
+
+);
+
+CREATE INDEX ON sandbox.token_balances_proposal_1 USING btree (address, contract_address);
+CREATE INDEX ON sandbox.token_balances_proposal_1 USING btree ( contract_address);
+
+
+DO $$
+DECLARE
+   hour_   text;
+   arr varchar[];
+BEGIN
+  select array(select generate_series('2021-03-21', '2021-03-23', '1 hour'::interval)::timestamptz)
+    into arr;
+   FOREACH hour_  IN   array arr
```

Comment:
Can this `DO` just be a `SELECT token_balances.insert_hourly(start_ts, end_ts)`?

Reply:
Makes sense, I thought along the same lines.